interview-questions.md

```diff
@@ -8,50 +8,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: statement based on cirtain condition met.
 
-Researched answer:
+Researched answer: they are a sequence of well defined instructions the produce a unique output based on the value of input. 
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Global information tracker.
 
 Researched answer:
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: conditional
 
-Researched answer:
+Researched answer: The if statement evaluates the Boolean value of true, the corresponding code statement will be excuted.
 
 3. What is an index? What is the difference between index and value?
 
-Your answer:
+Your answer: location of character or value. 
 
-Researched answer:
+value is contents of elements in an array 
 
-4. What is iteration?
+Researched answer: Every value in the array has a particula location known as the index. 
 
-Your answer:
+4. What is iteration? repeating function solving process.
+
+Your answer: repeating process of performing tasks
+
+Researched answer: It is the process of excuting a block of code over and over again until a condition met. 
 
-Researched answer:
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-Your answer:
+Your answer: navigator and driver to improve problem solving in coding.
 
-Researched answer:
+Researched answer: Two people collaborating on one code source to collectively produce one program. 
 
 ## Looking Ahead: Terms for Next Week
 
```

Comment:
It's always great to look ahead at the terms so you have some context when hearing the terms during lectures.